pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -49,79 +49,4 @@ If you're already using RSpec for your tests, add the `rspec-buildkite-analytics
     $ git push
     ```
 
-5. Make sure that the Test Analytics [environment variables](#environment-variables) are set so that the RSpec collector can use them in your Test Analytics dashboard.
```

Comment:
One less thing to do! 🚀

Reply:
About this one -- I think we *do* need something on the individual collector pages that help people understand more about env vars. The biggest customer support reach out now is people believing that TA is broken, when really they're missing env vars.

(And then there's the group of people that don't even bother reaching out to customer support, and I just happen to notice that things aren't working right when I look at their setups. They're just sitting there thinking that Test Analytics sucks 😆)

So, while TA will technically work with these instructions (ie, you will see graphy type things in the UI) using only the one required var, if the magic doesn't work (which is not uncommon on first setup because containers), TA will break -- ie, parallel runs won't de-dupe, etc -- and then people either give up on Test Analytics, or contact support.

🤔 Maybe it's a final step that's like "confirm things are working properly"?

🤔 Also, part of the solution here may be to change this sentence:
`Add the Test Analytics code to your application in spec/spec_helper.rb, and set the environment variable [securely](http://localhost:3000/docs/pipelines/secrets) on your agent or agents.`

"The" env var, to me, implies that there are no more to set, when in reality, there are -- even if we're setting them magically, there's more going on behind the scenes. 🤔

Reply:
I think I understand the container things now that I've written about it. 😅 

I've partly addressed this with ba2bd0278a2a78134521a6a00104ca0425693175, but it would be nice to have the post-procedure success crtieria (though not strictly blocking).

Reply:
My understanding is that it is as simple as folks not seeing corect links to their CI builds (in whatever supported CI) because those vars weren't available when they expected them to be, because they weren't passed through to the container running the collector.